ide/db/src/org/netbeans/modules/db/explorer/action/RefreshAction.java

```diff
@@ -35,6 +40,25 @@
  *
  * @author Rob
  */
+@ActionRegistration(
+        displayName = "#Refresh", 
+        lazy = false,
+        enabledOn = @ActionState(type = DatabaseConnection.class, useActionInstance = true)
+)
+@ActionID(category = "Database", id = "netbeans.db.explorer.action.Refresh")
+@ActionReferences(value = {
+    @ActionReference(path = "Databases/Explorer/Connection/Actions", position = 100),
```

Comment:
The action had originally different positions in the `Databases/Explorer/**/Actions" - is the reorder OK now ? It seems that it fit in between some separator-divided groups with the original position = 350 (?)

Reply:
Fixing in 40a21f6